src/util/fields.py

```diff
@@ -196,6 +197,7 @@ def remove_field(cr, model, fieldname, cascade=False, drop_column=True, skip_inh
     :param bool drop_column: whether the field's column is dropped
     :param list(str) or str skip_inherit: list of inheriting models to skip the removal
                                           of the field, use `"*"` to skip all
+    :param bool keep_attachments: for binary fields, whether the data should be kept as attachments.
```

Comment:
Ideally we should align the text to the max column (not done below due to limitations of GH review textarea)
```suggestion
    :param bool keep_as_attachments: for binary fields, whether the data should be kept as attachments
```